src/plugins/console/server/routes/api/console/proxy/create_handler.ts

```diff
@@ -130,7 +136,7 @@ export const createHandler = ({
     });
   } catch (e: any) {
     const isResponseErrorFlag = isResponseError(e);
-
+    if (!isResponseError) log.error(e);
```

Comment:
here looks like - if not error, log error?

Reply:
if response error, we return to client side, don't log
If it's not response error, we add a log here for troubleshoot